test/requests.test.js

```diff
@@ -242,6 +242,40 @@ describe("narratives", () => {
   }
 });
 
+describe("groupOverview", () => {
+
+  describe("groups", () => {
+    testPaths([
+      { case: "overview", path: "/groups/test/settings/overview" },
+    ]);
+  });
```

Comment:
These tests now fail after rebasing to include d36f8441e0fd7d9d288cd92aa898caeb356b8a9d and use the `authz.tags.Type.Group` on the group overview endpoint, without conditionally adding `authz.tags.Visibility.Public` here:

https://github.com/nextstrain/nextstrain.org/blob/d36f8441e0fd7d9d288cd92aa898caeb356b8a9d/src/groups.js#L112-L114

`Visibility.Public` seems intended for datasets/narratives, so this is probably fine. I'll remove these tests given manual tests cover the same thing with authz.

Reply:
Nevermind. After inspecting failing tests, I think adding `Visibility.Public` to the Group's authz policy is needed if we want to `setGroup()` on the base endpoint `/groups/:groupName`:

https://github.com/nextstrain/nextstrain.org/blob/69ae64418f0a2e5ab3da03e364de0382ca3a9603/src/app.js#L286-L287

since that still calls `assertAuthorized()` on the `Group`:

https://github.com/nextstrain/nextstrain.org/blob/69ae64418f0a2e5ab3da03e364de0382ca3a9603/src/endpoints/groups.js#L8-L11

Reply:
> since that still calls `assertAuthorized()` on the `Group`:

Ah, [it doesn't](https://gist.github.com/tsibley/a42514eca2a44a73e5bf7eb33cfe3235#file-nopaste-diff-L105-L115) in [the patch I pasted](https://github.com/nextstrain/nextstrain.org/pull/570#discussion_r957687615). (Though that patch was off-the-cuff so may not be fully baked in other ways.)

Reply:
I see, somehow I missed that while splitting your patch up. I added that one line change in 055efa733fdf52ea237b11f43198345afb242831 with my understanding of its purpose in the commit message.